app-constants.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+require("dotenv").load();
+
+const kEnvironmentVariables = [
+  "SERVER_URL",
+  "PORT",
+  "COOKIE_SECRET",
+  "DEBUG_DUMMY_SMTP",
+  "SMTP_HOST",
+  "SMTP_PORT",
+  "SMTP_USERNAME",
+  "SMTP_PASSWORD",
+  "OAUTH_AUTHORIZATION_URI",
+  "OAUTH_TOKEN_URI",
+  "OAUTH_PROFILE_URI",
+  "OAUTH_CLIENT_ID",
+  "OAUTH_CLIENT_SECRET",
+];
+
+var AppConstants = {
+  init() {
+    for (let v of kEnvironmentVariables) {
+      if (process.env[v] === undefined) {
+        throw "Required environment variable was not set: " + v;
```

Comment:
nit: We should throw an error:

```js
throw new Error(`Required environment variable was not set: ${v}`);
```

Also, should `DUMMY_DEBUG_SMTP` be required, or can we safely just assume the default is `false` unless specified?

Reply:
Yeah I think we can make `DUMMY_DEBUG_SMTP` a special case actually. I expect to remove it at some point.